functions/package.json

```diff
@@ -16,9 +17,6 @@
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage"
   },
-  "engines": {
```

Comment:
Why do you want to remove this protection?

Reply:
I took inspiration of another repo pour you are right! I'm putting it back :)

Reply:
As per: https://github.com/algolia/firestore-algolia-search/blob/main/functions/package.json I added a limit on node v15 and plus